projects/plugins/jetpack/modules/google-fonts/wordpress-6.3/constants.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Define the constants of the Jetpack Google Fonts module.
+ *
+ * @package automattic/jetpack
+ */
+
+/**
+ * Curated list of Google Fonts
+ * See https://wp.me/p9Jlb4-22P
```

Comment:
Since we're making changes, we may as well fix that. Let's avoid direct links like that:

```suggestion
 * See p9Jlb4-22P-p2
```